apps/script/deploy-storybook.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+echo "Going to build and deploy storybook to cdo-styleguide gh-pages repo"
+
+# get rid of old build if it is still around for some reason
+rm -rf storybook-deploy
+
+echo "Fetching application.css from localhost, I hope you have dashboard running"
+
+# fetch version of application.css that is generated by rails
+curl -o build/package/css/application.css http://localhost-studio.code.org:3000/assets/application.css
+
+echo "Building the static storybook site"
+# build the static storybook site
+build-storybook -o storybook-deploy -s build/package/
+cd storybook-deploy
+
+echo "Removing some unused files"
+# remove a bunch of crap we don't actually need.
+rm -rf media
+mv js/en_us/common_locale.js .
+rm -rf js
+mkdir -p js/en_us
+mv common_locale.js js/en_us/
+
+echo "Pushing to github"
+# create the git repo and push to gh-pages
+git init
+git add .
+git commit -m "initial commit"
+git remote add origin git@github.com:code-dot-org/cdo-styleguide.git
```

Comment:
I just created this repo to go with the diff. In trying to answer the question of "what's the easiest way to host some static files somewhere" I decided to just use gh-pages. Since I didn't want to mess up any existing gh-pages branch of an existing repo, I just created a new repo which only has a gh-pages branch.